CRM/Utils/Check/Component/Env.php

```diff
@@ -116,6 +116,27 @@ public function checkPhpMysqli() {
   }
 
   /**
+   * @return array
+   */
+  public function checkPhpEcrypt() {
+    $messages = array();
+    $test_pass = 'iAmARandomString';
+    $encrypted_test_pass = CRM_Utils_Crypt::encrypt($test_pass);
+    if ($encrypted_test_pass == base64_encode($test_pass)) {
+      $messages[] = new CRM_Utils_Check_Message(
+        __FUNCTION__,
+        ts('Your PHP does not include the recommended encryption functions. Some passwords will not be stored encrypted, and if you have recently upgraded from a PHP that does include these functions, your encrypted passwords will not be decrypted correctly. If you are using PHP 7.0 or earlier, you probably want to include the "%1" extension.'
+          array('1' => 'mcrypt'),
```

Comment:
Comma on `128` is missing, and comma on `129` is extra.

For whatever reason, PHP array+function notation are slightly with respect to trailing commas. (`array(1,2,)` is OK but `myfunc(1,2,)` is not.)